src/logcollector/logcollector.c

```diff
@@ -1826,15 +1827,24 @@ void * w_output_thread(void * args){
                     if(sleep_time < 300)
                         sleep_time += 5;
                 }
-            } else {
-                merror(QUEUE_SEND);
+            }
+            
+        } else {
+            int messageSent = 0;
+            while (messageSent <= 0) {
```

Comment:
Now, when `SendMSGtoSCK` return `< 0` the thread is locked until success. Is that the expected behavior?